i18n/zh-CN/docusaurus-plugin-content-docs/version-2.1/data-operate/import/import-way/insert-into-values-manual.md

```diff
@@ -0,0 +1,309 @@
+ EtlFinishTime: 2024-11-20 16:44:08
+ LoadStartTime: 2024-11-20 16:44:08
+LoadFinishTime: 2024-11-20 16:44:08
+           URL: 
+    JobDetails: {"Unfinished backends":{},"ScannedRows":0,"TaskNumber":0,"LoadBytes":0,"All backends":{},"FileNumber":0,"FileSize":0}
+ TransactionId: 61071
+  ErrorTablets: {}
+          User: root
+       Comment: 
+1 row in set (0.00 sec)
+```
+
+### 取消导入作业
+
+用户可以通过 Ctrl-C 取消当前正在执行的 INSERT INTO VALUES 作业。
+
+## 参考手册
+
+### 导入命令
+
+INSERT INTO VALUES 一般仅用于 Demo，不建议在生产环境使用。
+
+```sql
+INSERT INTO target_table (col1, col2, ...)
+VALUES (val1, val2, ...), (val3, val4, ...), ...;
+```
+
+### 导入配置参数
+
+**01 FE 配置**
+
+**insert_load_default_timeout_second**
+
+- 默认值：14400（4 小时）
+
+- 参数描述：导入任务的超时时间，单位：秒。导入任务在该超时时间内未完成则会被系统取消，变成 CANCELLED。
+
+**02 环境变量**
+
+**insert_timeout**
+
+- 默认值：14400（4 小时）
+
+- 参数描述：INSERT INTO VALUES 作为 SQL 语句的的超时时间，单位：秒。
+
+**enable_insert_strict**
+
+- 默认值：true
+
+- 参数描述：如果设置为 true，当 INSERT INTO VALUES 遇到不合格数据时导入会失败。如果设置为 false，INSERT INTO VALUES 会忽略不合格的行，只要有一条数据被正确导入，导入就会成功。
+
+- 解释：2.1.4 版本及以前，INSERT INTO VALUES 无法控制错误率，只能通过该参数设置为严格检查数据质量或完全忽略错误数据。常见的数据不合格的原因有：源数据列长度超过目的数据列长度、列类型不匹配、分区不匹配、列顺序不匹配等。
+
+**insert_max_filter_ratio**
+
+- 默认值：1.0
+
+- 参数描述：自 2.1.5 版本。仅当 `enable_insert_strict` 值为 false 时生效。用于控制 `INSERT INTO VALUES` 时的错误容忍率。默认为 1.0 表示容忍所有错误。可以取值 0 ~ 1 之间的小数。表示当错误行数超过该比例后，INSERT 任务会失败。
+
+### 导入返回值
+
+INSERT INTO VALUES 是一个 SQL 语句，其返回结果会包含一个 JSON 字符串。
+
+其中的参数如下表说明：
+
+| 参数名称 | 说明                                                         |
+| -------- | ------------------------------------------------------------ |
+| Label    | 导入作业的 label，通过 INSERT INTO tbl WITH LABEL label ... 指定 |
+| Status   | 表示导入数据是否可见。如果可见，显示 `visible`，如果不可见，显示 `committed`<p>- `visible`：表示导入成功，数据可见</p> <p>- `committed`：该状态也表示导入已经完成，只是数据可能会延迟可见，无需重试</p> <p>- Label Already Exists：Label 重复，需要更换 label</p> <p>- Fail：导入失败</p> |
+| Err      | 导入错误信息                                                 |
+| TxnId    | 导入事务的 ID                                                |
+
+**INSERT 执行成功**
+
+```sql
+mysql> INSERT INTO test_table (user_id, name, age) VALUES (1, "Emily", 25), (2, "Benjamin", 35), (3, "Olivia", 28), (NULL, "Alexander", 60), (5, "Ava", 17);
+Query OK, 5 rows affected (0.05 sec)
+{'label':'label_26eebc33411f441c_b2b286730d495e2c', 'status':'VISIBLE', 'txnId':'61071'}
+```
+
+其中 `Query OK` 表示执行成功。`5 rows affected` 表示总共有 5 行数据被导入。
+
+**INSERT 执行成功但是有 warning**
+
+```sql
+mysql> INSERT INTO test_table (user_id, name, age) VALUES (1, "Emily", 25), (2, "Benjamin", 35), (3, "Olivia", 28), (NULL, "Alexander", 60), (5, "Ava", 17);
+Query OK, 4 rows affected, 1 warning (0.04 sec)
+{'label':'label_a8d99ae931194d2b_93357aac59981a18', 'status':'VISIBLE', 'txnId':'61068'}
+```
+
+其中 `Query OK` 表示执行成功。`4 rows affected` 表示总共有 4 行数据被导入。`1 warnings` 表示被过滤了 1 行。
+
+当需要查看被过滤的行时，用户可以通过 [SHOW LOAD](../../../sql-manual/sql-statements/Show-Statements/SHOW-LOAD)语句。返回结果中的 URL 可以用于查询错误的数据，具体见后面 查看错误行 小结。
```

Comment:
New path : ../../../sql-manual/sql-statements/data-modification/load-and-export/SHOW-LOAD.md